setup/src/Magento/Setup/Model/Grid/Module.php

```diff
@@ -178,7 +178,7 @@ private function getModulesInfo(array $moduleList)
     private function addGeneralInfo(array $items)
     {
         foreach ($items as &$item) {
-            $item['moduleName'] = $item['moduleName'] ?: $this->packageInfo->getModuleName($item['name']);
+            $item['moduleName'] = isset($item['moduleName']) ? $item['moduleName'] : $this->packageInfo->getModuleName($item['name']);
```

Comment:
The line length exceeds 120 characters. According to PSR-2 coding style, a line length must be  <= 120 characters. Also as an improvement, I would recommend using null coalescing operator to simplify the construction (http://php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce)